src/pages/en/guides/integrations/tailwind.md

```diff
@@ -0,0 +1,187 @@
+  integrations: [tailwind({
+    // Example: Provide a custom path to a Tailwind config file
+    config: { path: './custom-config.js' },
+  })],
+}
+```
+
+</details>
+
+<details>
+  <summary><strong>config.applyBaseStyles</strong></summary>
+
+  <br/>
+
+By default, the integration imports a basic `base.css` file on every page of your project. This basic CSS file includes the three main `@tailwind` directives:
+
+```css
+/* The integration's default injected base.css file */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+To disable this default behavior, set `config.applyBaseStyles` to `false`. This can be useful if you need to define your own `base.css` file (to include a [`@layer` directive](https://tailwindcss.com/docs/functions-and-directives#layer), for example). This can also be useful if you do not want `base.css` to be imported on every page of your project.
+
+```js
+// astro.config.mjs
+export default {
+  integrations: [tailwind({
+    // Example: Disable injecting a basic `base.css` import on every page.
+    // Useful if you need to define and/or import your own custom `base.css`.
+    config: { applyBaseStyles: false },
+  })],
+}
+```
+
+</details>
+
+## Examples
+
+*   The [Astro Tailwind Starter](https://github.com/withastro/astro/tree/latest/examples/with-tailwindcss?on=github) gets you up and running with a base for your project that uses Tailwind for styling
+*   Astro's homepage uses Tailwind. Check out its [Tailwind configuration file](https://github.com/withastro/astro.build/blob/main/tailwind.config.js) or an [example component](https://github.com/withastro/astro.build/blob/main/src/components/integrations/IntegrationCard.astro)
+*   The [Astro Ink](https://github.com/one-aalam/astro-ink), [Sarissa Blog](https://github.com/iozcelik/SarissaBlogAstroStarter), and [Creek](https://github.com/robertguss/Astro-Theme-Creek) themes use Tailwind for styling
+*   [Browse Astro Tailwind projects on GitHub](https://github.com/search?q=%22%40astrojs%2Ftailwind%22+filename%3Apackage.json\&type=Code) for more examples!
+
+## Troubleshooting
+
+*   If your installation doesn't seem to be working, make sure to restart the dev server.
+*   If you edit and save a file and don't see your site update accordingly, try refreshing the page.
+*   If you edit and save a file and don't see your site update accordingly, try refreshing the page.
+*   If refreshing the page doesn't update your preview, or if a new installation doesn't seem to be working, then restart the dev server.
+
+For help, check out the `#support-threads` channel on [Discord](https://astro.build/chat). Our friendly Support Squad members are here to help!
+
+You can also check our [Astro Integration Documentation][astro-integration] for more on integrations.
+
+[astro-integration]: https://docs.astro.build/en/guides/integrations-guide/
+
+[astro-ui-frameworks]: https://docs.astro.build/en/core-concepts/framework-components/#using-framework-components
```

Comment:
There may not be anything we can do about these, but note that these are now hard-coded, full URLs because they originate externally.

I can't imagine it being a practical problem, but it *did* throw me for a loop when I clicked and expected to still be in the Netlify preview (light mode) and not taken externally to the live docs site (dark mode).  😄

Reply:
I thought I had those fixed (they cause the link checker to complain). I'll get on that.

Reply:
Fixed in [37e48f0](https://github.com/withastro/docs/pull/954/commits/37e48f0c408d61f6d51daa6537f8a040edac7825)